orchestra/views.py

```diff
@@ -187,3 +193,49 @@ def submit_task_assignment(request):
 # A simple status endpoint for things like health checks, etc.
 def status(request):
     return HttpResponse('OK')
+
+
+class OrchestraRegistrationView(RegistrationView):
```

Comment:
is this impl essentially the same as RegistrationView except we do `signals.orchestra_user_registered.send` at the end?  if so, can we make our implementation call the parent class' implementation?